mitodl/mitxpro
# Zagaran/boeing voucher

#### What are the relevant tickets?
This PR adds the boeing_voucher app as requested by Ferdi and Peter.

#### What's this PR do?
This PR creates the boeing_voucher app. Since we do not have access to any voucher pdfs that do not contain sensitive data, we can't commit any tests at this time.

#### How should this be manually tested?
A reviewer should open a python terminal, load a relevant voucher pdf as a file, and run `pdf_parsing.py's ` `read_pdf` function on the pdf file.

#### Where should the reviewer start?
A lot of the code was autogenerated for a standard Django app. The reviewer should focus on pdf_parsing.py for now.
